lib/draft/output_vacation_distribution_file.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Draft.OutputVacationDistribution do
+  @moduledoc """
+  Create a csv file containing the given employee vacation assignments in the hastus-required format.
+  """
+  alias Draft.EmployeeVacationAssignment
+
+  @spec output_vacation_distribution_file([Draft.EmployeeVacationAssignment.t()], String.t()) ::
+          [String.t()]
+  def output_vacation_distribution_file(vacation_assignments, output_file_path) do
+    {:ok, output_file} = File.open(output_file_path, [:write])
+
+    formatted_data = Enum.map(vacation_assignments, &EmployeeVacationAssignment.to_csv_row(&1))
+
+    :ok =
+      File.write(
+        output_file_path,
```

Comment:
```suggestion
        output_file,
```

Or you can write to `output_file_path` directly without opening/closing the file explicitly.